data/applications/mlpipelines.yaml

```diff
@@ -0,0 +1,17 @@
+metadata:
+  name: mlpipelines
+  annotations:
+    opendatahub.io/categories: 'ML Pipelines'
+spec:
+  displayName: ML Pipelines
+  provider: Kubeflow
+  description: |-
+    Pipeline tool specialized for Machine Learning workloads, leveraging Kubeflow and Openshift Pipelines
+  kfdefApplications: ['ml-pipelines']
+  route: ml-pipeline-ui
+  img: images/kubeflow.svg
+  category: Red Hat managed
+  support: red hat
```

Comment:
This should be changed to to reflect that it is an ODH Core component and the support is from the ODH community directly.  `category` is up for discussion because I can't remember what we wanted it to signify in the Dashboard

```
category: ODH Core
support: Open Data Hub
```